bodhi-server/bodhi-server.spec

```
@@ -75,6 +83,10 @@ Requires: skopeo
 The Bodhi composer is the component that publishes Bodhi artifacts to
 repositories.
 
+%if 0%{?fedora} && 0%{?fedora} >= 41
```

```suggestion
%if 0%{?fedora} >= 41
```

Also, consider making this a `%bcond libdnf5` instead, so you don't have to repeat the version all over the spec.